src/xunit.performance.api/Profilers/Etw/EtwModule.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
+{
+    /// <summary>
+    /// Loaded module for the corresponding process.
+    /// </summary>
+    public sealed class EtwModule
+    {
+        /// <summary>
+        /// The fully qualified name of the module file.
+        /// </summary>
+        public string FileName { get; set; }
+
+        /// <summary>
+        /// File name and extension of the module.
+        /// </summary>
+        public string ImageName => Path.GetFileName(FileName);
```

Comment:
Any reason to have both the FileName and Image name?  I'd think you'd want to have the full path to the file, and then you can cut that down whenever you need to.